Shared/ViewModels/LibraryViewModel.swift

```diff
@@ -94,10 +94,14 @@ final class LibraryViewModel: ViewModel {
 			genreIDs = filters.withGenres.compactMap(\.id)
 		}
 		let sortBy = filters.sortBy.map(\.rawValue)
+		let queryRecursive = filters.filters.contains(.isFavorite) ||
+			self.person != nil ||
+			self.genre != nil ||
+			self.studio != nil
 
 		ItemsAPI.getItemsByUserId(userId: SessionManager.main.currentLogin.user.id, startIndex: currentPage * pageItemSize,
 		                          limit: pageItemSize,
-		                          recursive: true,
+		                          recursive: queryRecursive,
```

Comment:
This needs to be a customization option whether it should always be `true` or with this flag, as people organize their media within a "Movies"/other folder on their systems and want all their items to be flattened.

Reply:
Even then, the web UI doesn't group folder items so it's good that we're doing something even better.

Have a toggle that belongs in settings under "Customization". If you don't know what I'm talking about, I can get this done.

Reply:
oh okay, I actually just implemented that as an additional filter option before seeing your reply. But I agree putting it as a global option might be better so I'll change it.